books/free-programming-books-langs.md

```diff
@@ -1289,6 +1289,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 #### Node.js
 
 * [An Introduction to libuv](https://nikhilm.github.io/uvbook/) - Nikhil Marathe  ([PDF](http://nikhilm.github.io/uvbook/An%20Introduction%20to%20libuv.pdf) - [ePub](http://nikhilm.github.io/uvbook/An%20Introduction%20to%20libuv.epub))
+* [Data brokering with Node.js: Process data at the speed of technology](https://heynode.com) *(Sign up requested)*
```

Comment:
Shouldn't this be in courses?

Reply:
looks like a tutorial, maybe you are right.
It's similar to:
https://github.com/EbookFoundation/free-programming-books/blob/8e452285166810c3eaf30ec3f192f89937ecdb5f/courses/free-courses-en.md#L314

https://github.com/EbookFoundation/free-programming-books/blob/8e452285166810c3eaf30ec3f192f89937ecdb5f/courses/free-courses-en.md#L908

Reply:
In any case there are not books because some of listed tutorials have videos as part of content, and thus you can't preserve the esence of content when printing

I'll opt with interactive tutorials

Reply:
> An interactive website which lets the user type code or commands and evaluates the result 

There are tutorials not interactive exercises, the "courses" section I think is more accurate

Reply:
let's put it in courses

Reply:
@xuandai-py